model/Check/System/AwsRDSFreeSpaceCheck.php

```diff
@@ -171,12 +168,35 @@ protected function getInstanceData(): ?array
 
         $InstanceData = null;
         foreach ($dbInstances['DBInstances'] as $dbInstance) {
-            if (strpos($dbInstance['DBInstanceIdentifier'], $stackId) === 0) {
+            if (str_starts_with($dbInstance['DBInstanceIdentifier'], $stackId)) {
```

Comment:
Application should still run with PHP 7.4, I believe `str_starts_with` only comes with php 8

Reply:
fixed